### PR TITLE
feat: Use a number as a shortcut for breakpoints without modifiers

### PR DESCRIPTION
**Pull request by kof**
## Description

Both ctrl/cmd + number are used to in browsers to switch between tabs. Webflow is just using a number shortcut without modifier, lets do the same

## Steps for reproduction

1. hit numbers to switch breakpoints
2. can still type numbers in inputs

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/shared/commands-emitter.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const findCommandsMatchingHotkeys = (event: KeyboardEvent) => {
     pressedKeys.add("shift");
   }
   if (altKey) {
-    pressedKeys.add("altKey");
+    pressedKeys.add("alt");
```
Comment (kof) [inline apps/builder/app/shared/commands-emitter.ts:60]: drive-by fix, I think we are not using alt yet, but it was inconsistent with others

File: apps/builder/app/builder/shared/commands.ts [hunk 2]
```diff
@@ -30,7 +30,8 @@ const makeBreakpointCommand = <CommandName extends string>(
   number: number
 ): Command<CommandName> => ({
   name,
-  defaultHotkeys: [`meta+${number}`, `ctrl+${number}`],
+  defaultHotkeys: [`${number}`],
+  disableHotkeyOnFormTags: true,
```
Comment (TrySound) [inline apps/builder/app/builder/shared/commands.ts:34]: this too
disableHotkeyOnContentEditable: true,

File: apps/builder/app/shared/commands-emitter.ts (hunk 1 above)
Comment (TrySound) [reply apps/builder/app/shared/commands-emitter.ts:60]: good catch

**Review by TrySound — APPROVED** [review]
